tracer/test/test-applications/integrations/Samples.HttpMessageHandler/Program.cs

```diff
@@ -69,7 +69,7 @@ public static async Task Main(string[] args)
                 }
 #endif
 
-#if NET5_0
+#if NET5_0 || NET6_0
```

Comment:
Shall we embrace these:
```suggestion
#if NET5_0_OR_GREATER
```

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/1885/commits/242c3c813e05991fdf1543aa6f7574ce577c6805